scripts/deploy/deployThirdParty.ts

```diff
@@ -60,39 +52,39 @@ async function main() {
 
   const network = NETWORKS[(process.env['NETWORK'] || 'LOCALHOST') as NETWORKS]
   if (!network) {
-    throw ('Invalid network')
+    throw 'Invalid network'
   }
 
   // Deploy the TPR contract
-  const ThirdPartyRegistry = await ethers.getContractFactory("ThirdPartyRegistry")
-  const tpr = await ThirdPartyRegistry.deploy(owner,
+  const ThirdPartyRegistry = await ethers.getContractFactory(
+    'ThirdPartyRegistry'
+  )
+  const tpr = await ThirdPartyRegistry.deploy(
+    owner,
     THIRD_PARTY_AGREGATOR[network],
     COLLECTOR[network],
     COMMITTEE[network],
-    MANA[network],
-    TIERS[network])
+    MANA[network]
```

Comment:
WE should add the oracle and item price here but using new envs variables.

Moreover we must create a deployChainlinkOracle.ts